services/app-api/src/domain-models/contractAndRates/statusType.ts

```diff
@@ -7,9 +7,15 @@ const statusSchema = z.union([
     z.literal('RESUBMITTED'),
 ])
 
+const reviewStatusSchema = z.union([
```

Comment:
In my mind there was a difference in verbiage but I don't see why it couldn't be consolidated

Reply:
I spoke with @macrael and he had a stronger opinion about keeping them separate because actions are verbs and the status are adjectives and they won't always match up (added a new example to the [miro](https://miro.com/app/board/o9J_lS5oLDk=/?moveToWidget=3458764577353419403&cot=14))